src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProjectCreationInfo.cs

```diff
@@ -13,6 +12,7 @@ internal class ProjectSystemProjectCreationInfo
         public CompilationOptions? CompilationOptions { get; set; }
         public string? FilePath { get; set; }
         public ParseOptions? ParseOptions { get; set; }
+        public string? IntermediateOutputPath { get; set; }
```

Comment:
See other comment: rename this to match whatever we called it on ProjectSystemProject.